client/src/components/App.js

```diff
@@ -28,7 +29,9 @@ const App = (props) => {
     <Router>
       <TopBar user={currentUser} />
       <Switch>
-        <Route exact path="/" component={ShoesListPage}/>
+        <Route exact path="/" component={ShoesListPage} />
+				<Route exact path="/shoes" component={ShoesListPage} />
+				<Route exact path="/shoes/new" component={NewShoeForm} />
```

Comment:
this indentation is weird, lets fix it

Reply:
It only looks like that on github, it looks normal in VScode not sure why its doing that